src/components/ListItems.js

```diff
@@ -1,55 +1,68 @@
 import React from 'react';
-import { Col, Row, Container, Input, InputGroup, Button, InputGroupAddon } from 'reactstrap';
+import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { BottonText, ListItem_stings } from './constants';
-import Row4 from './rows/row4';
-import Row3 from './rows/row3';
-import Row2 from './rows/row2';
-import Row1 from './rows/row1';
+import { Layout, Row, Col, Icon } from 'antd';
+import { ListItem_stings } from './constants';
+import SearchItems from './SmallerComponents/Search';
```

Comment:
there should not be a SmallerComponents folder. This does not conform with our coding conventions

Reply:
Removed